src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig

```diff
@@ -714,3 +724,68 @@
    </span>
   </div>
 {% endblock generatable_text_widget %}
+
+{% block change_password_widget %}
```

Comment:
I'm not sure we're going to reuse this widget 🤔

Reply:
you never know :smile: 
however, is it bad to have it as a widget?

Reply:
I'm thinking maybe we have a premature abstraction usecase here 😉.

I suggest we follow DRY and [WET](https://dev.to/wuz/stop-trying-to-be-so-dry-instead-write-everything-twice-wet-5g33) principles here and wait for a need to reuse it before creating a widget.

This `bootstrap_4_layout.html.twig` file contains highy-reusable and generic widgets such as `translatable_widget` so putting these quite specific widgets next to them feels wrong to me 🤔 

Moreover this file and its neighbors (`form_div_layout.html.twig` for example) could use some refactoring as they grew bigger and bigger overtime. So let's not make them even bigger if not needed 😛